app/views/NiphlNumberView.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
```

Comment:
I think this file might need formatting with line indents

Reply:
updated this now